src/ua/hillel/Dog.java

```diff
@@ -1,4 +1,4 @@
-public class Dog extends Animal {
+public class  Dog extends Animal {
     private int maxRun = 500;
     private int maxSwim = 10;
     private static int DogCount;
```

Comment:
Class' members should have a specific order:

constants
static variables
instance variables
constructors
getters + setters
public methods
private methods
Please change this class' members order according to the info in this comment.